build/Linux+BSD/org.musescore.MuseScore.appdata.xml.in

```diff
@@ -267,40 +255,13 @@
         </ul>
       </description>
     </release>
-
-    <release version="2.2.1" date="2018-04-03"/>
-    <release version="2.2" date="2018-03-26"/>
-    <release version="2.1.0" date="2017-05-02"/>
+    <release date="2018-04-03" version="2.2.1"/>
+    <release date="2018-03-26" version="2.2"/>
+    <release date="2017-05-02" version="2.1.0"/>
   </releases>
-  
-  <content_rating type="oars-1.1">
```

Comment:
I agree with @mirabilos: there doesn't seem to be any disadvantage to providing these content warnings.

Reply:
No longer an empty tag, shouldn’t cause problems now.